lib/astutils.cpp

```diff
@@ -341,7 +342,68 @@ bool isReturnScope(const Token * const endToken)
     return false;
 }
 
-bool isVariableChanged(const Token *start, const Token *end, const unsigned int varid)
+bool isVariableChangedByFunctionCall(const Token *tok, const Settings *settings, bool & inconclusive)
```

Comment:
hmm.. it's not necessary with the Settings. The Library is enough. On the other hand it's shorter to access the settings in the function calls. What do you think?